android-sample/src/main/java/com/zachklipp/richtext/sample/MarkdownSample.kt

```diff
@@ -145,39 +174,73 @@ private fun CheckboxPreference(
 
 private val sampleMarkdown = """
   # Demo
-  Based on [this cheatsheet][cheatsheet]
+  Based on [cheatsheet](google.com)
 
   ---
 
-  ## Headers
-  ---
-  # Header 1
-  ## Header 2
-  ### Header 3
-  #### Header 4
-  ##### Header 5
-  ###### Header 6
-  ---
+  ## Links
+
+  blah blah [I'm an inline-style link](https://www.google.com)
+
+  [I'm a relative reference to a repository file](../blob/master/LICENSE) hi hi
   
-  ## Full-bleed Image
-  ![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/Image_created_with_a_mobile_phone.png/1920px-Image_created_with_a_mobile_phone.png)
+  Autolink option will detect text links like https://www.google.com and turn them into Markdown links automatically.
 
-  ## Images smaller than the width should center
-  ![](https://cdn.nostr.build/p/4a84.png)
+  [I'm an inline-style link](https://www.google.com) test akjasd 
```

Comment:
Do you want to either revert this or update it so that you're only inserting one block of text so if we ever need to rebase our fork, there will be fewer conflicts?

Reply:
yeah i'll revert the text